packages/frontend/core/src/pages/workspace/detail-page/detail-page.css.ts

```diff
@@ -22,7 +22,6 @@ export const affineDocViewport = style({
   flexDirection: 'column',
   userSelect: 'none',
   containerName: 'viewport',
-  // TODO(@JimmFly): find out what this does in bs
```

Comment:
so, found out?

Reply:
This style itself is copied from [block-suite-editor/styles.css.ts](https://github.com/toeverything/AFFiNE/commit/f832ada98cf8fcf09339bd3a2c08cb95c95f5016#diff-30ec7d1597a28e385440da4a95099eb4ae0dc7a828c2b0bd246cfbb41f642872), and the style inside is completely different.
Regarding this Class Name, there are many places in blocksuite to query it. This is a required ClassName

Reply:
It is to determine the main doc padding width on the left & right.